modules/loadFromStorage.js

```diff
@@ -0,0 +1,15 @@
+import storageAvailable from './storageAvailable.js';
+import populateBook from './populateBook.js';
+
+const loadFromStorage = () => {
+  if (storageAvailable('localStorage')) {
+    const { localStorage } = window;
+    const bookData = JSON.parse(localStorage.getItem('bookList'));
+    if (bookData !== null) {
+      // bookList = bookData;
```

Comment:
- [ ] Please, consider also removing not used/commented part of your code, to make it clear 👍